app/src/main/java/io/homeassistant/companion/android/settings/qs/views/ManageTilesView.kt

```diff
@@ -154,6 +155,17 @@ fun ManageTilesView(
                         }
                     }
                 }
+
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Text(
+                        text = stringResource(R.string.tile_vibrate),
+                        fontSize = 15.sp
+                    )
+                    Checkbox(

```

Comment:
Use a `Switch`, checkboxes are for selecting [one or more in a list](https://m3.material.io/components/checkbox/overview)